src/sonic-config-engine/minigraph.py

```diff
@@ -532,9 +532,9 @@ def parse_dpg(dpg, hname):
                 intfs_inpc.append(pcmbr_list[i])
                 pc_members[(pcintfname, pcmbr_list[i])] = {}
             if pcintf.find(str(QName(ns, "Fallback"))) != None:
-                pcs[pcintfname] = {'members': pcmbr_list, 'fallback': pcintf.find(str(QName(ns, "Fallback"))).text, 'min_links': str(int(math.ceil(len() * 0.75))), 'lacp_key': 'auto'}
```

Comment:
>members [](http://example.com/codeflow?start=36&length=7)

Do you want to update src/sonic-yang-models/yang-models/sonic-portchannel.yang at the same time?

@ganglyu for vis.

Reply:
Yes, removed from the yang file.